### ISSUE TITLE
Should the connect function return a PyClient or a Connection?

### ISSUE THREAD

**Issue by bielim** [hannesmuehleisen/MonetDBLite-Python#30]
As a new user, I wanted to familiarize myself with MonetDBLite-Python by working through the basic commands explained [here](https://www.monetdb.org/blog/monetdblite-for-python).

I used the following commands to create a connection:
```python
import monetdblite
conn = monetdblite.connect('/path/to/database')
```

where `/path/to/database` is an empty directory.
I expected conn to be a `Connection` object, but my attempt to create a cursor using
```python
c = conn.cursor()
```
resulted in an `AttributeError` ('PyClient' object has no attribute 'cursor').

The definition of the `connect` function in  #init.py reads as follows:
```python
def connect(*args, **kwargs):                                                   
    #return Connection(*args, **kwargs)                                          
    return embeddedmonetdb.connect()
```
Thus, it doesn't return a `Connection` object but the `PyClient` object created by `embeddedmonetdb.connect()`. This `PyClient` object is generated without any arguments, which made me doubt if it is really the intended return value of connect. Things seem to work if I uncomment the first return statement and comment out the second one. Is there a reason why a `PyClient` object should be returned instead of a `Connection` object?

Corresponding pull request: #29

**Comment by kutsurak** [comment]
Hi @bielim,

You are absolutely right that `monetdblite.connect()` should return a `Connection` object. In fact this is one of the fixes that will be part of the 0.7.0 version (take a look at https://github.com/hannesmuehleisen/MonetDBLite-Python/commit/d7b85142c80dcc5143d0d1ad2744c70ace519850), along with greater compliance with [PEP-249](https://www.python.org/dev/peps/pep-0249/). This release is planned for sometime early in Q2 2019.

In the meantime, you can use the `monetdblite.make_connection()`, that indeed returns a `Connection` object.

Best regards,
Panos.

**Comment by bielim** [comment]
Thanks, @kutsurak!
I'm looking forward to the 0.7.0 version - until then, using ```monetdblite.make_connection()``` solves my problem.

Best,
Melanie
